integration-cli/docker_cli_help_test.go

```diff
@@ -1,12 +1,13 @@
 package main
 
 import (
-	"os"
 	"os/exec"
 	"runtime"
 	"strings"
 	"testing"
 	"unicode"
+
+	"github.com/docker/docker/pkg/homedir"
 )
 
 func TestMainHelpWidth(t *testing.T) {
```

Comment:
@duglin yeah i get the same thing on master (and this PR fixes it), however this test still passes on master.